source/index.html.md

```diff
@@ -686,7 +686,8 @@ object][ApplePay-PaymentToken] for more information.
     <span class="type">[0-9]{12,19}</span>
   </dt>
   <dd>
-    Primary account number of card to charge.
+    Primary account number of card to charge.<br />
+    If <code>cryptogram</code> parameter is included, the <code>pan</code> must be a Token PAN.
```

Comment:
```suggestion
    Primary account number (PAN) of card to charge.<br />
    If <code>cryptogram</code> parameter is included, the PAN must be a Token PAN.
```

I notice we use PAN in many places without having introduced it, and we also stubbornly write "primary account number" in a few places. Perhaps we ought to introduce PAN earlier and then only use PAN after that point.